reporter/src/main/kotlin/Reporter.kt

```diff
@@ -22,6 +22,9 @@ package org.ossreviewtoolkit.reporter
 import org.ossreviewtoolkit.model.AnalyzerResult
 import org.ossreviewtoolkit.model.OrtResult
 import org.ossreviewtoolkit.model.ScanRecord
+import org.ossreviewtoolkit.model.config.PathExclude
```

Comment:
Commit message nit: just "...to deduplicate them."

Reply:
Done.